src/components/Room/GameRoom.tsx

```diff
@@ -197,38 +197,42 @@ const GameRoom: React.FunctionComponent = () => {
 
     return (
         <div className={styles.container}>
-            <h2 className={styles.title}>{renderTitleContent()}</h2>
-            <List className={styles.list_container}>
-                {players.map((player) => (
-                    <ListItemText
-                        key={player.name}
-                        primary={
-                            <span
-                                style={
-                                    player.name === myName
-                                        ? { fontWeight: 'bold' }
-                                        : undefined
-                                }
+            <div className={styles.wrapper}>
+                <div className={styles.title}>{renderTitleContent()}</div>
+                <div className={styles.list_container}>
+                    {players.map((player) => (
+                        <div className={styles.list_container_item}>
+                            <div
+                                className={styles.player_avatar}
+                                style={{
+                                    backgroundImage: `url(https://avatars.dicebear.com/api/micah/${player.name}.svg)`,
+                                }}
+                            />
+                            <div
+                                className={`${styles.player_name} ${
+                                    player.name === myName ? styles.player_name_self : ''
+                                }`}
```

Comment:
Maybe can use `cssUtils.joinClassName` for this

Reply:
Okay